Source/Extensions/Blazorise.Exporters/Targets/File/IFileExportOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Blazorise.Exporters;
+
+/// <summary>
+/// Defines configuration options for exporting content to a file, including file name, extension, and MIME type.
+/// </summary>
+public interface IFileExportOptions
+{
+    /// <summary>
+    /// Constructs the full file name by combining the file name without extension and the file extension.
+    /// </summary>
+    public string FileName => $"{FileNameNoExtension}.{FileExtension}";
```

Comment:
Does it really matters that we need too have separate props for `FileName`, `FileNameNoExtension`, `FileExtension`? I think just one, `FileName` is enough. If the user provides the filename without extension, it's OK. If the user user provides extension it's still good. Either way I don't think it matters.

We can always have a default extension applied internally if the user doesn't specify it.

Reply:
Yes, that is an option. But my taking is this, we only need one `FileName` property. And prevent users from confusing them. Should they use only one, all of them, or a combination of different properties? 

Better just give them one to use.

Reply:
Much better now. Thanks.